modules/API/API.js

```diff
@@ -1355,6 +1357,19 @@ class API {
         });
     }
 
+    /**
+     * Notify external application (if API is enabled) wether the used browser is supported or not.
+     *
+     * @param {boolean} supported - If browser is supported or not.
+     * @returns {void}
+     */
+    notifyBrowserSupport(supported: boolean) {
+        this._sendEvent({
+            name: 'browser-support',
```

Comment:
Don't we want a constant defined in external_api.js and a method to check it?

Reply:
Yeah, I guess that's a better way. It did not cross my mind 😊

Reply:
@saghul Tried to add a method but although it looks like we have tree shaking the bundle for external_api increases 10 fold to 300k. I don't this is something we want...

Reply:
Sure